FrontEnd/src/tests/CompanyCard.test.js

```diff
@@ -0,0 +1,151 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import CompanyCard from '../components/SearchPage/search_field/companies/CompanyCard';
+
+afterEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('CompanyCard component unit tests', () => {
+  test('renders years of experiense', () => {
+    const company = {
+      id: 1,
+      name: 'Testname',
+      categories: [1, 2],
+      region: 'Testregion',
+      founded: 2005,
+      service_info: 'Testinfo',
+      address: 'Testadress',
+      banner_image: '',
+    };
+
+    render(
+      <MemoryRouter>
+        <CompanyCard isAuthorized={{ isAuth: true }} companyData={company} />
+      </MemoryRouter>
+    );
+    const divElement = screen.getByText(/18 років досвіду/i, { exact: false });
+    expect(divElement).toBeInTheDocument();
+  });
+
+  test('testing stars', () => {
+    const company = {
+      id: 1,
+      name: 'Testname',
+      categories: [1, 2],
+      region: 'Testregion',
+      founded: 2005,
+      service_info: 'Testinfo',
+      address: 'Testadress',
+      banner_image: '',
+    };
+    jest.mock('axios');
+    const axios = require('axios');
+
+    async () => {
+      await axios.get.mockResolvedValue({
```

Comment:
Do you need async/await here?
I think we can import axios once and then mock it rather than importing it for every test

Reply:
Thank you for feedback. Values, returned by axios.get.mockResolvedValue, are different. So I can't mock it once 
Deleted async/await